src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -231,6 +231,12 @@ pub enum Connector {
     AvroOcf {
         path: String,
     },
+    /// S3 Objects
+    S3 {
+        bucket: String,
+        /// A glob-like pattern that objects must match
+        objects_pattern: String,
+    },
```

Comment:
How would you feel about:

```markdown
A glob-like object specifier: `'a/**/*.json'`
```

This seems to match what happens in syn, in e.g. Item: https://docs.rs/syn/1.0.57/syn/enum.Item.html

Reply:
I still think that's a bit too specific since the syntax of that pattern is opaque to the SQL parser. Like I think the correct level of detail for these doc comments is something like

```
The `S3 ...` connector.
S3 {
    /// The `BUCKET 'bucket'` clause.
    bucket: String,
    /// The `OBJECTS 'pattern'` clause.
    object_patterns: String,
}
```

And then the semantics of these things would be fully-described in the user-facing docs, so that we focus our documentation efforts on what users will actually see. But again this is super minor so :100: down to roll with what you have!